sys-fs/btrfsmaintenance/btrfsmaintenance-0.3.1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils
+
+DESCRIPTION="Scripts for btrfs maintenance tasks like periodic scrub, balance, trim or defrag"
+HOMEPAGE="https://github.com/kdave/btrfsmaintenance"
+GITTAG="v${PV}"
+SRC_URI="https://github.com/kdave/btrfsmaintenance/archive/${GITTAG}.tar.gz -> btrfsmaintenance-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	virtual/cron
+	sys-fs/btrfs-progs"
+
+src_configure() {
+	sed -i -e 's#/etc/sysconfig/btrfsmaintenance#/etc/conf.d/btrfsmaintenance#' *.sh
+}
+
+src_install() {
+	dodoc README.md CONTRIBUTING.md btrfsmaintenance.changes
+	newconfd sysconfig.btrfsmaintenance btrfsmaintenance
+	insinto /usr/share/btrfsmaintenance
+	doins btrfsmaintenance-functions
+	exeinto /usr/share/btrfsmaintenance
+	doexe btrfs*.sh
+	insinto /usr/lib/systemd/system
+	doins btrfsmaintenance-refresh.service
+}
+
+pkg_postinst() {
+	elog "Now edit cron periods and mount points in /etc/conf.d/btrfsmaintenance"
+	elog "then run /usr/share/btrfsmaintenance/btrfsmaintenance-refresh-cron.sh to"
+	elog "update cron symlinks"
+}
+
+pkg_config() {
+	elog "Installing default btrfsmaintenance scripts"
+	"${ROOT}"/usr/share/btrfsmaintenance/btrfsmaintenance-refresh-cron.sh
+}
+
+pkg_prerm() {
```

Comment:
@mgorny The workflow of this package is as follows: you can configure the time schedule of the maintenance scripts in the config file. The "/usr/share/btrfsmaintenance/btrfsmaintenance-refresh-cron.sh" then creates symlinks in the cron.daily / cron.weekly / cron.monthly directories to the worker scripts. So  btrfsmaintenance-refresh-cron.sh must be run at least once to give the whole package a working default configuration. After this, the btrfsmaintenance-refresh-cron.sh can be run any number of times - without changing the configuration it does not change any symlinks.
Finally, before removing the package, the symlinks in cron.[daily|weekly|monthly] must be removed, otherwise they point to non-existing destinations.
In the g-o DevManual is no hint that pkg_prerm is called before every removal. If this is the case, it would be nice to find a hint there.
Is there a way (variable or something) I can check inside pkg_prerm if the call is during a package upgrade or final remove?